proposals/smi.adoc

```diff
@@ -0,0 +1,123 @@
+== Service Mesh Interface Proposal
+*Name of Project:*
+Service Mesh Interface
+
+*Project Description:*
+Service Mesh Interface is an open source specification for interoperable service meshes on Kubernetes. Launched in May 2019, SMI brings the ecosystem together with a standard which enables innovation by providing a common abstraction that anyone can build against.
+
+*Statement on alignment with CNCF charter mission:*
+The Service Mesh Interface spec is well-aligned with the CNCF mission to make cloud native computing ubiquitous. Interoperability creates a virtuous cycle which increases adoption. SMI encourages an open source, vendor-neutral service mesh ecosystem by empowering end users to build upon ecosystem tools without needing to re-implement the underlying substrate.
+
+*Comparison with similar projects (inside or outside the CNCF), including what differentiates this project:*
+SMI is complementary to existing CNCF projects like LinkerD; it encourages adoption of service mesh. SMI is the only cross-industry effort to make service meshes interoperable. For example, LinkerD is one example of a service mesh that implements and uses SMI.
```

Comment:
Are there any other examples besides linkerd? I recall there being a large cabal of companies who support it.